pkgs/servers/home-assistant/default.nix

```
@@ -45,10 +45,14 @@ let
       });
     };
 
-  py = python3.override {
-    # Put packageOverrides at the start so they are applied after defaultOverrides
-    packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) ([ packageOverrides ] ++ defaultOverrides);
-  };
+  py = python3.override(oa: {
```

This is not how `override` works, is it? It doesn't expect a function.

---

Oh, after reading `lib/customisation.nix` it seems like this is possible:
https://github.com/NixOS/nixpkgs/blob/8e84cafcd9350ce11e3f630003b859d4c70a483a/lib/customisation.nix#L67-L93
(pay attention to the `if` in the definition of `overrideWIth`)
But then I think we should use
```suggestion
  py = python3.override (oldAttrs: {
```
because that's common naming.

---

Not actually ´oldArgs`?

---

True, that makes even more sense.